### PR TITLE
Enhancement/cohtml panel UI

### PR DESCRIPTION
**Pull request by alien1976**
Task -> https://coherent-labs.atlassian.net/browse/COH-17450
PR in the devtools repo -> https://github.com/CoherentLabs/cohtml/pull/2193

### CODE REVIEW

**Review by palikar — APPROVED** [review]

File: front_end/entrypoints/inspector_main/CohtmlPanel.ts [hunk 1]
```diff
@@ -171,78 +192,25 @@ export class CohtmlPanelView extends UI.Widget.VBox implements SDK.TargetManager
     this.continuousRepaintSetting = Common.Settings.Settings.instance().moduleSetting('continuousRepaint');
 
     this.contentElement.classList.add('cohtml-panel-base');
+    this.controlsPanel = this.contentElement.createChild('div');
+    this.controlsPanel.classList.add('cohtml-panel');
+    this.controlsPanel.classList.add('controls-panel');
```
Comment (venetak) [inline front_end/entrypoints/inspector_main/CohtmlPanel.ts]:
```suggestion
    this.controlsPanel.classList.add('cohtml-panel', 'controls-panel');
```

File: front_end/entrypoints/inspector_main/CohtmlPanel.ts [hunk 2]
```diff
@@ -171,78 +192,25 @@ export class CohtmlPanelView extends UI.Widget.VBox implements SDK.TargetManager
...
 
-    const controlsPanel = this.contentElement.createChild('div');
-    controlsPanel.classList.add('controls-panel');
+    this.sidebar = this.contentElement.createChild('div');
+    this.sidebar.classList.add('cohtml-sider');
+    this.addSidebarEvents();
 
     this.contentPanel = this.contentElement.createChild('div');
+    this.contentPanel.classList.add('cohtml-panel');
     this.contentPanel.classList.add('content-panel');
```
Comment (venetak) [inline front_end/entrypoints/inspector_main/CohtmlPanel.ts]:
```suggestion
    this.contentPanel.classList.add('cohtml-panel', 'content-panel');
```

File: front_end/entrypoints/inspector_main/CohtmlPanel.ts [hunk 3]
```diff
@@ -437,34 +521,39 @@ export class CohtmlPanelView extends UI.Widget.VBox implements SDK.TargetManager
...
+    const buttonItem = UI.UIUtils.createTextButton(i18nString(label), click, 'cohtml-button');
+    UI.ARIAUtils.markAsLink(buttonItem);
 
-    const descItem = newButtonBlock.createChild('span');
+    const descItem = newButtonBlock.createChild('div');
     descItem.classList.add('button-desc');
     descItem.innerText = desc;
+    newButtonBlock.appendChild(descItem);
+
+    if (subDesc) {
```
Comment (venetak) [inline front_end/entrypoints/inspector_main/CohtmlPanel.ts]:
```suggestion
    if (subDescription) {
```
? for consistency :D

**Review by venetak — APPROVED** [review]

**Review by Tony-CL — APPROVED** [review]

File: front_end/entrypoints/inspector_main/CohtmlPanel.ts (hunk 3 above)
Comment (alien1976) [reply front_end/entrypoints/inspector_main/CohtmlPanel.ts]: Fixed

File: front_end/entrypoints/inspector_main/CohtmlPanel.ts (hunk 2 above)
Comment (alien1976) [reply front_end/entrypoints/inspector_main/CohtmlPanel.ts]: Fixed

File: front_end/entrypoints/inspector_main/CohtmlPanel.ts (hunk 1 above)
Comment (alien1976) [reply front_end/entrypoints/inspector_main/CohtmlPanel.ts]: Fixed

**Pull request merged**